middleware/src/handlers/handlers.go

```diff
@@ -38,11 +38,12 @@ type Middleware interface {
 	ShutdownBase() rpcmessages.ErrorResponse
 	RebootBase() rpcmessages.ErrorResponse
 	EnableRootLogin(rpcmessages.ToggleSettingArgs) rpcmessages.ErrorResponse
+	EnablePasswordLogin(rpcmessages.ToggleSettingArgs) rpcmessages.ErrorResponse
```

Comment:
Can you rename this to `EnableSSHPaswordLogin`? I constantly need to remind myself it is just for ssh when I look at it.

Reply:
Fixed in https://github.com/digitalbitbox/bitbox-base/pull/237/commits/0d998b415de239f892e621d0e387cd15019b95b5